src/containers/SinglePositionWrapper/SinglePositionWrapper.tsx

```diff
@@ -63,10 +63,10 @@ export const SinglePositionWrapper: React.FC<IProps> = ({ id }) => {
 
   const [isFinishedDelayRender, setIsFinishedDelayRender] = useState(false)
 
-  const poolKey =position?  poolKeyToString(position?.poolKey) || '' : ''
+  const poolKey = position ? poolKeyToString(position?.poolKey) || '' : ''
```

Comment:
i think this can be replaced with `position ? poolKeyToString(position.poolKey) : ''`